python/gui/auto_generated/qgisinterface.sip.in

```diff
@@ -591,11 +591,7 @@ Returns the native Add Point Cloud Layer action.
 
     virtual QAction *actionAddAfsLayer() = 0;
 %Docstring
-Returns the native Add ArcGIS FeatureServer action.
-%End
-    virtual QAction *actionAddAmsLayer() = 0;
```

Comment:
This is technically an API break -- I'd make it just return the same generic ArcGIS REST server action instead of removing it.